deltachat-ios/Controller/Settings/AdvancedViewController.swift

```diff
@@ -320,7 +329,10 @@ internal final class AdvancedViewController: UITableViewController {
             Utils.authenticateDeviceOwner(reason: String.localized("pref_password_and_account_settings")) { [weak self] in
                 self?.showAccountSettingsController()
             }
-
+        case .proxySettings:
+            Utils.authenticateDeviceOwner(reason: String.localized("proxy_settings")) { [weak self] in
+                self?.showProxySettings()
+            }
```

Comment:
no need to do a device-owner-verification here, at least, we're also not doing that on android and also telegram does not do it.

it seems mostly important to have that dialog fast and flawlessly working, as proxy change, ppl that really need them often try out things in that dialog. all device-owner-verification will slow that down (some more (pin) and slome less (face-id) :)  

also, if someone adds a proxy without you to know, you will be able to notice that by the (upcoming) icon).

cc @adbenitez